.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: false
+language: node_js
+matrix:
+  fast_finish: true
+  include:
+    - node_js: "6"
+      env: TASK=browser
+cache:
+  directories:
+  - node_modules
+script:
+  - make ${TASK}
+env:
+  # See how to get encrypted env variables for saucelabs here:
+  # https://docs.travis-ci.com/user/sauce-connect/
+  global:
+    # travis encrypt SAUCE_USERNAME=futil
+    - secure: "intAImWiGH3IyrtSghpK8EarKekOCOtEClVQwkKKtPa4vyJ1AtxRfSbsSqaWP8XxDG24eqca6rEPBETrTALyQPNFZZGk/+9f5eO5fqm1BRV5aghEJdKmLVxi+iBn5VLnwru3EI11EnNMZBobMPjJDIiRo7lLRuMizSWy1koAHVjeNGu+UQWEafNcU0n/kja9/nmwidrB5dBrGRuazkkQSxIWxpi2SvZNPCI6tUKcVqldkn/jjlTDr01+BtRtVUtn8M2IsufB/+cXCtsSyBo3DQTzZof91LSxB5ywhTANipB2xP+6dGoUaPbuefxhfaHsnm9mkRuWyUrMLizQR/jbu5CR37Yh0VS29cjPFT3coWi3c7LdzfuahSAFF9KLT+66C/j7gnJ9UKPGSoW+B+sXq5t9T8rWBLe6gX3L13P6Xh1VtpSIc4OjPlKj0b5VahS49kDkgGhrcDUkkpVH/ajgCgN/ZVuc93jp1kvBitKJEGj0DRrk7Oe8mxpcDCelY0dasq4qg40A/C8X1T70lSbDbCiVe1sIChIe6Y/m49/QmnCEI3vjMbnXMQxP9gtFFHrTxu77Qi+vB2Rezmt9PvtrbPIucHZTB+mqdJEKF0aNrJduli3UhSK2bh96WUbDB4yGIIPz0HM2OEVh913T8qRckFmOLoxnPizwT9BqN1uHt5w="
```

Comment:
Hi! Silly question, but: having these here isn't a security issue? We can put them in travis rather soon if you want. I guess that would be bound to our saucelabs account or something 🤔

Reply:
That's a very good question.  Yes, the secures definitely could be put in the Travis project settings as environment variables.  Having the encrypted values in public Git should also be fine as only Travis has the private/decryption key (correct me if I was wrong).

And yes, you can replace them with your own saucelabs username & key using the command mentioned in the comment.

Reply:
@sadasant We'll probably be using our current CircleCI configuration for this and our own key. I think @lsm just did it in travis as a demonstration (thanks for that, btw!) 👍

Reply:
@sean-clayton No problem.  That's exactly what I was trying to do.  It seems the combination of `Localtunnel + Travis` doesn't work very well as I can't get it all passing even once after many retries.  The same tests do pass smoothly on my laptop.  Change to a paid plan/service may change the figure since both are free services.